aspnet/fundamentals/routing.rst

```diff
@@ -186,100 +370,56 @@ Adding a colon ``:`` after the name allows additional inline constraints to be s
     - Steve
     - Used to enforce that a non-parameter value is present during during URL generation.
 
-Inline constraints must match one of the above options, or an exception will be thrown.
-
-.. tip:: To constrain a parameter to a known set of possible values, you can use a regex: ``{action:regex(list|get|create)}``. This would only match the ``action`` route value to ``list``, ``get``, or ``create``. If passed into the constraints dictionary, the string "list|get|create" would be equivalent. Constraints that are passed in the constraints dictionary (not inline within a template) that don't match one of the known constraints are also treated as regular expressions.
+.. warning:: Route constraints that verify the URL can be converted to a CLR type (such as ``int`` or ``DateTime``) always use the invariant culture - they assume the URL is non-localizable.
```

Comment:
We should also add here that route constraints based on CLR types don't perform a conversion for you. The value in the route values will be a string, not an int or datetime. We've recently had a few questions about this.